lib/foreman_rh_cloud/engine.rb

```diff
@@ -231,4 +255,8 @@ def self.ca_cert
       ::SETTINGS[:ssl_ca_file]
     end
   end
+
+  def self.on_prem_smart_proxy_features
+    ['insights']
```

Comment:
Since users will see it in the UI, should it be this?
```suggestion
    ['red_hat_lightspeed']
```

Reply:
Will the on premise version also be called lightspeed?

Reply:
I'd keep it insights, as that would be the upstream name.

Another note, most of the features have first letter capitalized: -> `Insights`.